dependencies.sh

```diff
@@ -12,7 +12,8 @@ export RUST_G_VERSION=3.1.0
 
 #node version
 export NODE_VERSION=14
-export NODE_VERSION_PRECISE=14.16.1
+export NODE_VERSION_PRECISE=20.12.0
+export NODE_VERSION_COMPAT=14.16.1
```

Comment:
I think we should rename NODE_VERSION_PRECISE to be NODE_VERSION_LATEST or something like that given the new context we're using it in. PRECISE doesn't really make sense when the base `NODE_VERSION` is still `14`

Reply:
Can do

Reply:
not trying to be obtuse, "latest" has a different version in node, this is the LTS version and i don't want to confuse anyone familiar

Reply:
ah okay